webextension/background/takeshot.js

```diff
@@ -28,18 +28,18 @@ window.takeshot = (function () {
         });
       });
     }
-    catcher.watchPromise(capturePromise.then(() => {
+    return catcher.watchPromise(capturePromise.then(() => {
       return uploadShot(shot);
     }).then(() => {
       let id = makeUuid();
-      clipboard.copy(shot.viewUrl);
       chrome.notifications.create(id, {
         type: "basic",
         iconUrl: "img/clipboard-32.png",
         title: "Link Copied",
         message: "The link to your shot has been copied to the clipboard"
```

Comment:
I feel like this sentence should have a period after it, but it's probably not a big deal.